hystrix-core/src/main/java/com/netflix/hystrix/HystrixThreadPool.java

```diff
@@ -227,8 +227,13 @@ private void touchConfig() {
                             dynamicCoreSize + " and maximumSize = " + configuredMaximumSize + ".  Maximum size will be set to " +
                             dynamicMaximumSize + ", the coreSize value, since it must be equal to or greater than the coreSize value");
                 }
-                threadPool.setCorePoolSize(dynamicCoreSize);
-                threadPool.setMaximumPoolSize(dynamicMaximumSize);
+                if (dynamicMaximumSize > dynamicCoreSize) {
+                    threadPool.setMaximumPoolSize(dynamicMaximumSize);
+                    threadPool.setCorePoolSize(dynamicCoreSize);
+                } else {
+                    threadPool.setMaximumPoolSize(dynamicMaximumSize);
+                    threadPool.setCorePoolSize(dynamicCoreSize);
+                }
```

Comment:
What's the difference between the `if` and the `else` part? Looks same to me. So if this really works in both cases, I'd change it to ...
```
threadPool.setMaximumPoolSize(dynamicMaximumSize);
threadPool.setCorePoolSize(dynamicCoreSize);
```
:slightly_smiling_face:

Reply:
For the secons statement: Yes.
The second statement is different (dynamicCoreSize vs. dynamicMaximumSize).
Could be written with ternary operator.